server/controllers/stock/reports/stock/entry_purchase_receipt.js

```diff
@@ -74,7 +75,10 @@ async function stockEntryPurchaseReceipt(documentUuid, session, options) {
     voucher_reference     : voucherReference,
   };
 
-  data.rows = rows;
+  data.rows = rows.map(row => {
+    row.expires = row.expires === 1;
```

Comment:
I think you are trying to make this into a boolean.  You can do that directly in the SQL:

```sql
IF(eg.expires, TRUE, FALSE) as expires
```

See if that works.  If it does, let's save `for` loop by doing this everywhere you need it.

Reply:
ah okay I didn't see this recommendation. thanks